### DISCUSSION TITLE
ERROR kaspa_miner] No workers specified

### DISCUSSION THREAD

**Discussion by ljburna** [tmrlvi/kaspa-miner#46] [General]
I am attenpting to runs the miner in Linux and the following message apprears

ERROR kaspa_miner] No workers specified

GPU: RX6700XT
OS: Fedora 36 
Kaspa Miner Version:kaspa-miner-v0.2.1-GPU-0.7-default-linux-gnu-amd64

I used the help guide to see if there is a parameter to add the worker and no parameter exist

![Selection_004](https://user-images.githubusercontent.com/113790414/190868278-ff042e8a-b47a-4a32-8081-89bfb20fba51.png)

**Comment by tmrlvi** [comment]
This is probably since opencl doesn't see your card (or that the name of the platform doesn't contain AMD)
Can you post the output of `clinfo -l`?

**Reply by dideldei** [reply]
I have the same issue and clinfo gives back 0 platforms.

**Reply by tmrlvi** [reply]
In this case, seems you are missing the opencl drivers. What is you device? AMD gpu?

**Reply by ravin-bhambhu07** [reply]
@tmrlvi i am facing the same issue and I am running it on a CPU, any suggestion how to resolve it?

**Reply by tmrlvi** [reply]
> @tmrlvi i am facing the same issue and I am running it on a CPU, any suggestion how to resolve it?

If you want to run on CPU, add `-t` and then the number of CPU workers you want. It is set to 0 by default in the GPU miner
